<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(130, 130, 130);
background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ABONNEMENT</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajout ou Modifier Abonnement </string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>751</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero l abonnement</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>116</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Libelle de l abonnement</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>180</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nombre des points</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID Client</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="numab">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="libab">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>120</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nbrab">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>180</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajouterabo">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>60</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>AJOUTER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modifierabo">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>150</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>MODIFIER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="quitterabo">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>230</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>QUITTER</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxab">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>250</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="refrechabo">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>250</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>330</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Date expiration de l'abonnement</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="datea">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>330</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher les abonnements</string>
       </attribute>
       <widget class="QTableView" name="tababo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>821</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero a rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_rech">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="rech_num">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher Numero</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID a rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_rechab">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="rech_id">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>80</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche ID client</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trie_asc_ab">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>120</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE ASC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trie_desc_abo">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE DESC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitterab">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Supprimer abonnement</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>641</width>
          <height>361</height>
         </rect>
        </property>
        <property name="title">
         <string>SUPPRISSION</string>
        </property>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero  a Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sup_abo">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>230</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>SUPPRIMER</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxsupprimer">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>160</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_sup">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>160</y>
           <width>20</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Imprimer les abonnements</string>
       </attribute>
       <widget class="QPushButton" name="imprimercl_2">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="impcc_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>851</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CLIENT</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajout ou Modifier client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>751</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>116</y>
           <width>56</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>180</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>PRENOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>EMAIL</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_aj">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nom_aj">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>120</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="prenom_aj">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>180</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="email_aj">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>250</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajoutcl">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>60</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>AJOUTER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modifcl">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>150</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>MODIFIER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="quittercl">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>230</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>QUITTER</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Afficher les clients</string>
       </attribute>
       <widget class="QTableView" name="tabclient">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>821</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID a rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_rech">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom a rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom_rech">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>80</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche NOM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trie_asc">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>120</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE ASC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trie_desc">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIEE DESC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitter_af">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Supprimer client</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>551</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>SUPPRISSION</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>40</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID a supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="suppcl">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>200</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>SUPPRIMER</string>
         </property>
        </widget>
        <widget class="QComboBox" name="combosup_cl">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="refrech">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>110</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Imprimer les clients</string>
       </attribute>
       <widget class="QPushButton" name="imprimercl">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="impcc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>851</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
